docs/Repository/README.md

```diff
@@ -0,0 +1,46 @@
+## Garden Linux Repositories
+
+Garden Linux is a Debian-based Operating System. Therefore it uses apt.
+
+The location of sources are at `/etc/apt/sources.list` respectively in the folder `/etc/apt/sources.list.d/`
+
+#### Default Repo:
+	deb http://repo.gardenlinux.io/gardenlinux dev main
+
+#### Generally:
+
+	deb http://repo.gardenlinux.io/gardenlinux $GARDENLINUX_EPOCH main
+
+`$GARDENLINUX_EPOCH` is generally `dev`, but the Garden Linux Version number can be defined here. It is the number of days since first Garden Linux release.
```

Comment:
I think we need more explanation here
here is my input:

---

`$days_since_garden_linux_release` is the number of days since 31.03.2020. 

For each day since we switched to an own package repository, we offer a respective distribution in our repo.gardenlinux.io apt repository. 

While `dev` contains always the latest package versions, the dated-distros contain the packages from that day. 

---